samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,142 @@
+.. _at_client_sample:
+
+nRF9160: AT Client
+##################
+
+The AT Client sample demonstrates the asynchronous serial communication taking place over UART to the nRF9160 modem.
+The sample enables you to use an external computer or MCU to send commands to the LTE-M/NB-IoT modem of your nRF9160 DK.
+
+Overview
+********
+
+The AT Client sample acts as a proxy for sending directives to the nRF9160 modem via AT commands.
+This facilitates the reading of responses or analyzing of events related to the nRF9160 modem.
+The commands can be initiated from a terminal or the `LTE Link Monitor`_, which is an application implemented as part of `nRF Connect for Desktop`_.
+
+For more information on the AT commands, see the `AT Commands Reference Guide`_.
+
+Also see :ref:`modem_info_readme` for information about how an LTE application can interact with the modem.
+
+
+Requirements
+************
+
+* The following development board:
+
+  * nRF9160 DK board (PCA10090)
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/at_client`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+
+Connecting to the nRF9160 DK board with LTE Link Monitor
+========================================================
+
+To connect to the nRF9160 DK board with LTE Link Monitor, perform the following steps:
+
+#. Launch the LTE Link Monitor after programming the AT Client sample onto the nRF9160 DK board.
+#. Connect the nRF9160 DK board to the PC with USB.
+#. Power on the nRF9160 DK board.
+#. Click :command:`Select Device` and select the particular board device entry from the drop-down list in the LTE Link Monitor.
+#. Observe that initially the command :command:`AT+CFUN?` is automatically sent to the modem, which returns a value 4, indicating that the modem is in the offline mode.
+#. Observe that the LTE Link Monitor terminal display also shows :command:`AT+CFUN=1` indicating that the modem has changed to the normal mode.
+#. Perform a reset of the nRF9160 DK board by pressing the reset button, which will reboot the board and start the AT Client sample.
+
+
+Testing
+=======
+
+After programming the sample to your board, test the sample by performing the following steps:
+
+#. `Connect to the nRF9160 development board with LTE Link Monitor <Connecting to the nRF9160 DK board with LTE Link Monitor_>`_.
+#. Run the following commands from the LTE Link Monitor terminal:
+
+   * Enter the command :command:`AT+CFUN?`.
```

Comment:
Is it clear enough that the "." (point) at the end of all AT commands is not part of the command?
Maybe we could change a bit the style of those command texts and add a gray background?

Reply:
Updated the structure of the sentences  containing commands so as to avoid the '.' at the end as follows:

- Enter the command: **AT+CFUN?**